api/types.proto

```diff
@@ -277,6 +277,41 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum OnFailure {
+		PAUSE = 0;
```

Comment:
The current behavior is that a change to the service replaces the previous rollout. This PR doesn't change that aspect.